prusti-viper/src/encoder/encoder.rs

```diff
@@ -89,7 +89,7 @@ pub struct Encoder<'v, 'tcx: 'v> {
     type_cast_functions: RefCell<HashMap<(ty::Ty<'tcx>, ty::Ty<'tcx>), vir::Function>>,
     memory_eq_encoder: RefCell<MemoryEqEncoder>,
     fields: RefCell<HashMap<String, vir::Field>>,
-    snapshots: RefCell<HashMap<String, Box<Snapshot>>>, // maps predicate names to snapshots
+    pub snapshots: RefCell<HashMap<String, Box<Snapshot>>>, // maps predicate names to snapshots
```

Comment:
Why is this needed?

Reply:
Please use the getters: https://github.com/viperproject/prusti-dev/blob/d0b6d9f8a56111f7c4269799e65b89bff5f6e16a/prusti-viper/src/encoder/encoder.rs#L1039-L1049